api4/team_local.go

```diff
@@ -84,7 +84,7 @@ func localInviteUsersToTeam(c *Context, w http.ResponseWriter, r *http.Request)
 		}
 		auditRec.AddMeta("errors", errList)
 		if len(goodEmails) > 0 {
-			c.App.SendInviteEmails(team, "Administrator", "mmctl "+model.NewId(), goodEmails, *c.App.Config().ServiceSettings.SiteURL)
+			c.App.Srv().EmailService.SendInviteEmails(team, "Administrator", "mmctl "+model.NewId(), goodEmails, *c.App.Config().ServiceSettings.SiteURL)
```

Comment:
Maybe we should wrap this calls from the API layer from the app layer, to give this the right isolation.

Reply:
Of course, the same for any change in the `api4` layer.

Reply:
Anyway, we are already accessing to the Store from the API, so the isolation here is already broken.

Reply:
Maybe a future task can be make the `App.Srv` method private, to disallow any server access through the App, that means, the app can use the server, any service that receive the server can use the server, but nobody can access the server directly using the app.